internal/commands/app/initialize/inputs.go

```diff
@@ -0,0 +1,100 @@
+package initialize
+
+import (
+	"github.com/10gen/realm-cli/internal/cli"
+	"github.com/10gen/realm-cli/internal/cloud/realm"
+	"github.com/10gen/realm-cli/internal/terminal"
+
+	"github.com/AlecAivazis/survey/v2"
+)
+
+const (
+	fromApp      = "an existing app"
+	fromTemplate = "a template"
+)
+
+type from struct {
+	Type       string
+	GroupID    string
+	AppID      string
+	TemplateID string
+}
+
+type inputs struct {
+	AppData         cli.AppData
+	Project         string
+	From            string
+	FromType        string
+	Name            string
+	DeploymentModel deploymentModel
+	Location        location
+}
+
+func (i *inputs) Resolve(profile *cli.Profile, ui terminal.UI) error {
+	appData, appDataErr := cli.ResolveAppData(profile.WorkingDirectory)
+	if appDataErr != nil {
+		return appDataErr
+	}
+	if appData.Name != "" {
+		return errProjectExists{}
+	}
+
+	if i.From != "" {
+		i.FromType = determineFromType(i.From)
+
+		// TODO(REALMC-XXXX): implement Realm app templates
```

Comment:
can we get the realm ticket for this? 

also - is the commented out code below for that ticket implementation?

Reply:
commented out code is for the implementation, of which I'm not sure when it will actually be planned (I think "app templates" will be its own project/epic).

I mentioned this above, but I can rip all the commented code out if you feel strongly about that.  This was primarily to help prove to myself that this command structure _could_ support some flavor of a app/templates flag (see the spec conversation I linked above for more details there)

Reply:
same as above